sktime/forecasting/compose/_pipeline.py

```diff
@@ -19,7 +18,6 @@
 
 
 class TransformedTargetForecaster(
-    _OptionalForecastingHorizonMixin,
     _SktimeForecaster,
```

Comment:
I think you should also remove the `_SktimeForecaster`

Reply:
I updated the recipe in #955, I forgot to talk about `SktimeForecaster` there

Reply:
this still should be done, it´s not a tags issue

Reply:
noted